README.md

```diff
@@ -30,7 +30,7 @@ yarn create-plots
 ```
 
 ### 4. Run the web app
-First, create `.env` in the `packages/web` directory with [the values from the Notion documentation](https://citydao.notion.site/Env-variables-9d5d8c5874c2400cb2554964cb61a86f)
+First, create a new `.env` file in the `packages/react-app` directory by copying `example.env` with [the values from the Notion documentation](https://citydao.notion.site/Env-variables-9d5d8c5874c2400cb2554964cb61a86f)
```

Comment:
Can someone has rights to Notion check this document? https://citydao.notion.site/Env-variables-9d5d8c5874c2400cb2554964cb61a86f If it is not accessible, we need to create a new one and put the mapbox key in it. It is good to hide the key even it can be seen in deployed frontend network requests.